deployment-examples/aws/k8s/values.yaml

```diff
@@ -0,0 +1,31 @@
+export TAG="0.0.1"
+
+image:
+  repository: $AWS_ECR_PATH/$AWS_ECR_REPOSITORY:$TAG
+  tag: '0.0.1'
+  pullPolicy: 'Always'
+
+## edit those environment variables
+## ----------------------
+nonSensitiveEnvs:
+  CTP_PROJECT_KEY: $CTP_PROJECT_KEY
+  CTP_API_URL: 'https://api.europe-west1.gcp.commercetools.com'
+  CTP_AUTH_URL: 'https://auth.europe-west1.gcp.commercetools.com'
+
+sensitiveEnvs:
+  CTP_CLIENT_ID: $CTP_CLIENT_ID
+  CTP_CLIENT_SECRET: $CTP_CLIENT_SECRET'
+## ----------------------
+
+# When the pod is scheduled, the container is guaranteed the amount of
+# resources requested 512MB RAM. As we don't know how much memory the jobs will
+# need, we don't set the limit
+resources:
+  requests:
+    memory: '1024Mi'
+
+# specifies how to treat concurrent executions of a job created by this cron job
+concurrency: Forbid
+
+# Avoid scheduling with this hack: as there is no day like 31 Feb :)
```

Comment:
What do you mean by this?

Reply:
https://github.com/commercetools/commercetools-subscriptions/pull/14/commits/1bacd877790695539eab4e6ff448b02cd5c26812